mmv1/templates/terraform/examples/dataplex_datascan_primary.tf.erb

```diff
@@ -5,7 +5,7 @@ resource "google_dataplex_datascan" "<%= ctx[:primary_resource_id] %>" {
   display_name = "Test Datascan"
 
   data = {
-	entity = "projects/<%= ctx[:test_env_vars]['project_name'] %>/locations/us-central1/lakes/amandeep-dev-lake/zones/rawzone/entities/austin_311"
+	entity = "projects/dataplex-clouddq/locations/us-central1/lakes/amandeep-dev-lake/zones/rawzone/entities/austin_311"
```

Comment:
i'm not sure we want to make this change, we want to keep the `ctx[:test_env_vars]['project_name']` and probably do something similar with locations, lakes, zones, and entities. Does that make sense? I'm imagining

`"projects/<%= ctx[:test_env_vars]['project_name'] %>/locations/ctx[:test_env_vars]['region']/lakes/ctx[:vars]['lake']/zones/ctx[:vars]['zone']/entities/ctx[:vars]['entity']"`

where `region` is defined in the [`test_env_vars`](https://github.com/GoogleCloudPlatform/magic-modules/pull/7911/files#diff-c09f956b6a8f8ef1cd3b80fa98674163413600ce849e90d655cb342877b07314R37) and `lake`, `zone` and `entity` are defined nearby, but in a similar `vars` block.

Let me know if you have questions around that! Thanks!